src/components/fader/index.tsx

```diff
@@ -0,0 +1,124 @@
+import _ from 'lodash';
+import React, {useCallback} from 'react';
+import {StyleSheet} from 'react-native';
+import View from '../view';
+import Image from '../image';
+
+export enum FaderLocation {
+  LEFT = 'LEFT',
+  RIGHT = 'RIGHT',
+  TOP = 'TOP',
+  BOTTOM = 'BOTTOM'
+}
+
+export type FaderProps = {
+  /**
+   * Whether the fader is visible (default is true)
+   */
+  visible?: boolean;
+  /**
+   * The location of the fader (the image is different), defaults to Fader.location.RIGHT
+   */
+  location?: FaderLocation;
```

Comment:
What do you think of renaming to `position`? to be more consistent with other components (toast position, badge position, etc..)

Reply:
Changed